test/built-ins/Object/getOwnPropertyDescriptors/symbols-omitted.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2016 Jordan Harband. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Object.getOwnPropertyDescriptors does not include Symbol keys.
```

Comment:
Shouldn't this read, "Object.getOwnPropertyDescriptors *includes* Symbol keys." ?